city_scrapers/spiders/chi_ssa_16.py

```diff
@@ -0,0 +1,105 @@
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        # There was no variation in types of meetings, so this was applicable for all.
+        return "Greektown Special Service Area Tax Commission #16 Public Meeting"
+
+    def _parse_description(self, item):
+        # This extracts the useful text provide to describe the services provided by the SSA.
+        return " ".join(
+            item.xpath('//span[text()="Improving the Community"]/parent::h4/parent::div/p/text()'
+                       ).extract()
+        )
+
+    def _parse_classification(self, item):
+        return COMMISSION
+
+    def _parse_start(self, item):
+        # Extracted all alphanumeric characters from the text of each meeting,
+        # which contained the dates of the meeting, then split into list containing M, D, Y
+        date = re.sub(r"[^a-zA-Z0-9]+", " ", item.xpath("text()").get()).split()
+
+        # Extracted just alphabetical characters from month, then used built-in strptime function
+        # in datetime to parse numeric month from string
+        month = re.sub(r"[^a-zA-Z]+", " ", date[0])
+        month = datetime.datetime.strptime(month, "%B").month
+
+        # Extracted numbers from day and year entries of date list to remove any errant characters
+        # (i.e. one entry said January 28th where all others read January 28).
+        day = int(re.sub(r"[^0-9]+", " ", date[1]))
+        year = int(re.sub(r"[^0-9]+", " ", date[2]))
+
+        # According to accompying information, all meetings in 2016 occurred at 3:00 PM
+        # All other years occurred at 2:00 PM.
+        if (year == 2016):
+            return datetime.datetime(year, month, day, 15, 0)
+        return datetime.datetime(year, month, day, 14, 0)
+
+    def _parse_end(self, item):
+        # Meeting adjournment information contained in files present in Minutes documents
+        # attached to each meeting that are un-optimized scanned docs.
+        return None
+
+    def _parse_time_notes(self, item):
+        # No other details present
+        return ""
+
+    def _parse_all_day(self, item):
+        return False
+
+    def _parse_location(self, item):
+        # Meetings occurred at same location for all years documented
+        return {
+            "address": "306 S. Halsted St, 2nd Floor, Chicago, ILL 60661",
+            "name": "SSA #16 Office",
+        }
+
+    def _parse_links(self, item):
+        # Most entries contained a Minutes document, but some not yet posted
+        # or otherwise have not occurred.
+        # When no anchor tag found, returns empty JSON element.
+        if (item.xpath("a/@href").get() is None):
+            return [{"href": "", "title": ""}]
```

Comment:
If there aren't any links we want to return an empty list and not one with a dictionary of blank values, so this can just return `[]`